cinn/ir/ir_schedule.h

```diff
@@ -106,6 +106,13 @@ class IRSchedule {
   //! Get a block with the specific name.
   Expr GetBlock(const std::string& block_name) const;
 
+  /**
+   * \brief Get all the childblocks of specific Block stored in ModuleExpr.
+   * @param block The block we find childblock in.
+   * @return ChildBlocks of the block.
+   */
+  std::vector<Expr> GetChildBlocks(const Expr& block) const;
+
```

Comment:
```suggestion
   */
  std::vector<Expr> GetChildBlocks(const Expr& expr) const;

```
接口注释补充说明expr可为for或block

Reply:
do ne